ingestion_server/ingestion_server/indexer.py

```diff
@@ -191,7 +201,7 @@ def __init__(
     # Helpers
     # =======
 
-    def get_stat(self, name_or_alias: str) -> tuple[bool, bool, Union[str, list[str]]]:
+    def get_stat(self, name_or_alias: str) -> Stat:
```

Comment:
The place where this function is called can also use the `stat` object now instead of unpacking it into 3 separate vars!

Reply:
Cool, implemented in a595497.